Cubical/ZCohomology/MayerVietorisUnreduced.agda

```diff
@@ -83,45 +83,47 @@ module MV {ℓ ℓ' ℓ''} (A : Type ℓ) (B : Type ℓ') (C : Type ℓ'') (f :
   dHomHelperPath : (n : ℕ) (h l : C → coHomK n) (a : C) → I → I → coHomK (suc n)
   dHomHelperPath zero h l a i j =
     hcomp (λ k → λ { (i = i0) → lUnitₖ 1 (0ₖ 1) (~ j)
-                    ; (i = i1) → lUnitₖ 1 (0ₖ 1) (~ j)
-                    ; (j = i0) → +ₖ→∙ 0 (h a) (l a) (~ k) i
-                    ; (j = i1) → cong₂Funct (λ x y → x +[ 1 ]ₖ y) (Kn→ΩKn+1 0 (h a)) (Kn→ΩKn+1 0 (l a)) (~ k) i})
+                   ; (i = i1) → lUnitₖ 1 (0ₖ 1) (~ j)
+                   ; (j = i0) → +ₖ→∙ 0 (h a) (l a) (~ k) i
+                   ; (j = i1) → cong₂Funct (λ x y → x +[ 1 ]ₖ y)
+                                           (Kn→ΩKn+1 0 (h a)) (Kn→ΩKn+1 0 (l a)) (~ k) i})
           (bottom i j)
        where
        bottom : I → I → coHomK 1
        bottom i j = hcomp (λ k → λ { (i = i0) → lUnitₖ 1 (0ₖ 1) (~ j)
-                                   ; (i = i1) → cong (λ x → lUnitₖ 1 x (~ j)) (Kn→ΩKn+1 0 (l a)) k})
+                                   ; (i = i1) → lUnitₖ 1 (Kn→ΩKn+1 0 (l a) k) (~ j) })
```

Comment:
@aljungstrom : inlining uses of cong like this makes the code a bit faster, probably because Agda doesn't have to infer any arguments